src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -752,20 +753,16 @@ public void onSuccess(Boolean result) {
    * @return
    */
   private ListenableFuture<Boolean> initializeDraft() {
-    if (isMailToIntent()) {
```

Comment:
just removing this may be the cause of the regression.

Reply:
No, that's fine, it's not the cause of the regression. I don't remember the details of the regression anymore, since it's complicated enough that everytime I don't think about it for some months, I completely forget all the details. But the regression has something to do with sometimes starting the activity using startActivity instead of startActivityForResult.